.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: bash
+sudo: required
```

Comment:
The only reason we need sudo is because that submodule behaves badly. I'd much rather we have our own installation instructions that just drop the script in the user's home dir and then `chmod`'s locally. The only reason we need sudo is because they drop it in `/usr/bin`.